parser/src/parser/tests/evaluator_functions.rs

```diff
@@ -1,8 +1,10 @@
+use air_script_core::{Range, TraceBinding};
+
```

Comment:
nit: we can probably import these from `crate::ast` below, or at least remove the extra empty line